pkg/kube/vault/vault_test.go

```diff
@@ -10,23 +11,34 @@ import (
 	k8sfake "k8s.io/client-go/kubernetes/fake"
 )
 
-func TestCreateVault(t *testing.T) {
+type VaultTestCase struct {
+	name               string
+	namespace          string
+	err                bool
+	authServiceAccount string
+	secretsPathPrefix  string
+}
+
+type vaultGKETestCase struct {
```

Comment:
Normally this is defined inside of the test.

Reply:
OK